components/adminComponents/challengeComponents/ChallengeList.tsx

```diff
@@ -54,10 +54,12 @@ export default function ChallengeList({
     <div className="p-5">
       <DndContext sensors={sensors} collisionDetection={closestCenter} onDragEnd={handleDragEnd}>
         <SortableContext items={challenges} strategy={verticalListSortingStrategy}>
-          {challenges.map((challenge, idx) => (
+          {challenges.map(({ title, description, prizes }, idx) => (
             <ChallengeCard
-              challenge={challenge}
-              key={challenge.id}
+              key={idx}
+              title={title}
```

Comment:
Not a fan of us arbitrarily adding props when we haven't defined any for the component.

Reply:
removed it